packages/dds/tree/src/test/feature-libraries/editable-tree/editableTree.spec.ts

```diff
@@ -89,19 +90,97 @@ function buildTestPerson(): readonly [SchemaDataAndPolicy, PersonType] {
 }
 
 describe("editable-tree", () => {
-    it("proxified forest", () => {
+    it("can use `Object.keys` and `getOwnPropertyDescriptor` with EditableTree", () => {
         const [, proxy] = buildTestPerson();
-        assert.ok(proxy);
+        assert(isUnwrappedNode(proxy));
+
         assert.equal(Object.keys(proxy).length, 5);
-        assert.deepEqual(proxy[getTypeSymbol](undefined, false), personSchema);
-        assert.deepEqual(proxy.address[getTypeSymbol](undefined, false), addressSchema);
+        assert.deepEqual([...Object.keys(proxy)], ["name", "age", "salary", "friends", "address"]);
```

Comment:
there is also code in this file that uses Reflect.ownKeys. Currently that should return the same as Object.keys since all our own keys are also enumerable, but it might be good to test that here by calling Reflect.ownKeys as well and checking it also return the expected set.